src/main/java/org/example/examplefinalProject/entity/Student.java

```diff
@@ -17,15 +31,13 @@ public class Student {
     private String studentSurname;
 
     @ManyToOne
-    @JoinColumn(name = "class_room_id")
+    @JoinColumn(name = "classRoomId")
     private ClassRoom classRoom;
```

Comment:
nerozumiem tomu preco sa dookola menia tieto idcka, napr v tomto pripade to ani nesedi s tym ako sa meni sql script. Vo vysledku je uplne jedno ktore sa bude pouzivat ale mali by ste sa uz finalne dohodnut na tom ktore sa bude pouzivat.